### PR TITLE
Fix order form errors

### PR DESCRIPTION
**Pull request by bthaile**
https://github.com/AugurProject/augur/issues/1368

fixes for mock parity, no changes to behavior.

Screen Shot 2019-03-19 at 8 24 41 PM
Screen Shot 2019-03-19 at 8 42 13 PM
Screen Shot 2019-03-19 at 8 42 41 PM

### CODE REVIEW

File: src/modules/trading/components/trading--form/trading--form.styles.less [hunk 1]
```diff
@@ -136,24 +166,24 @@
...
     color: @color-white;
-    font-family: 'Roboto Mono', monospace;
+    font-family: "Roboto Mono", monospace;
     font-size: @font-size-medium;
 
     &::placeholder {
-      color: rgba(255, 255, 255, 0.2);
-      font-family: 'Roboto Mono', monospace;
+      color: fade(@color-secondary-text, 50%);
+      font-family: "Roboto Mono", monospace;
```
Comment (phoebemirman) [inline src/modules/trading/components/trading--form/trading--form.styles.less]: there is a constant for this font you should use

File: src/modules/trading/components/trading--form/trading--form.styles.less [hunk 2]
```diff
@@ -136,24 +166,24 @@
...
       font-size: @font-size-medium;
     }
   }
 
   &.error {
-    border: 0.0625rem solid rgba(255, 39, 39, 0.5) !important;
+    border: 0.0625rem solid fade(@color-error-text, 50%) !important;
```
Comment (phoebemirman) [inline src/modules/trading/components/trading--form/trading--form.styles.less]: should avoid or get rid of these !important things if possible

File: src/modules/trading/components/trading--form/trading--form.styles.less [hunk 3]
```diff
@@ -63,31 +77,47 @@
...
 
-  svg {
+  > svg {
     margin: 0 0.2rem;
-    min-height: 1.75em;
-    min-width: 1.75em;
-    vertical-align: middle;
+    min-height: 1rem;
+    min-width: 1rem;
+    vertical-align: text-bottom;
```
Comment (phoebemirman) [inline src/modules/trading/components/trading--form/trading--form.styles.less]: Screen Shot 2019-03-19 at 7 26 15 PM
i don't think this vertical alignment is working here. the text and svg are centered together. this is how it looks in the mock but probably figma being weird?

File: src/modules/trading/components/trading--form/trading--form.styles.less [hunk 4]
```diff
@@ -63,31 +77,47 @@
 }
 
 .TradingForm__error_message_container {
-  background: rgba(255, 39, 39, 0.2);
-  border: 1px solid rgba(255, 39, 39, 0.2);
+  background: @color-error-message-bright-fade;
+  border: 1px solid @color-error-text;
```
Comment (phoebemirman) [inline src/modules/trading/components/trading--form/trading--form.styles.less]: rem?

File: src/modules/trading/components/trading--confirm/trading--confirm.styles.less [hunk 5]
```diff
@@ -138,7 +138,7 @@
 
 .TradingConfirm__error_message_container {
   background: fade(@color-darker-teal, 90%);
-  border: 1px solid @color-bright-teal;
+  border: 1px solid @color-negative;
```
Comment (phoebemirman) [inline src/modules/trading/components/trading--confirm/trading--confirm.styles.less]: rem?

Comment (bthaile) [reply src/modules/trading/components/trading--confirm/trading--confirm.styles.less]: I don't see an alternative for 1px in the code.

**Comment by bthaile** [comment]
added nowrap, so ext doesn't break.

Screen Shot 2019-03-19 at 10 38 57 PM

File: src/modules/trading/components/trading--form/trading--form.styles.less (hunk 4 above)
Comment (bthaile) [reply src/modules/trading/components/trading--form/trading--form.styles.less]: I don't see an alternative for 1px in the code. Not sure what 1px is in rem.

File: src/modules/trading/components/trading--form/trading--form.styles.less (hunk 3 above)
Comment (bthaile) [reply src/modules/trading/components/trading--form/trading--form.styles.less]: error in UI looks fine to me. in the mock it's not quite lined up correctly. It's not bottom aligned and not center aligned. In UI I went with text-bottom aligned.

File: src/modules/trading/components/trading--form/trading--form.styles.less (hunk 2 above)
Comment (bthaile) [reply src/modules/trading/components/trading--form/trading--form.styles.less]: removed the importants.

File: src/modules/trading/components/trading--confirm/trading--confirm.styles.less (hunk 5 above)
Comment (phoebemirman) [reply src/modules/trading/components/trading--confirm/trading--confirm.styles.less]: 0.0625rem

**Comment by bthaile** [comment]
I'll look at the gas higher back ground on branch.

**Pull request merged**